schemas/styles/csl.rnc

```diff
@@ -387,21 +390,29 @@ div {
   names.name =
     element cs:name {
       name.attributes,
-      
-      ## Select the "long" (first name + last name, for Western names),
-      ## "short" (last name only, for Western names), or "count" name form
-      ## (returning the number of names in the name variable, which can be
-      ## useful for some sorting algorithms).
-      [ a:defaultValue = "long" ]
-      attribute form { "long" | "short" | "count" }?,
+      name.forms,
       affixes,
       
       ## Set the delimiter for names in a name variable (e.g., ", " in
       ## "Doe, Smith")
       [ a:defaultValue = ", " ] delimiter,
       font-formatting,
-      name.name-part*
+      name.name-part*,
+      name.name-type*
```

Comment:
Is it intended that `name-part` and `name-type` are siblings? I'm not sure, but couldn't it make sense to define `name-part` as a child element of `name-type`? Or to allow both variants.

```xml
<names variable="author">
  <name>
    <name-part name="family" text-case="uppercase"/>
  </name>
</names>
```

And:

```xml
<names variable="author">
  <name>
    <name-type name="personal" form="short">
      <name-part name="family" text-case="uppercase"/>
    </name-type>
  </name>
</names>
```

Reply:
Any comments about this question?